CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add group authorization to PUT/events/{event_id}/images/{image_id} endpoint.[#719](https://github.com/rokwire/rokwire-building-blocks-api/issues/719)
 - Add group authorization to DELETE /events/{event_id}/images/{image_id} endpoint.[#720](https://github.com/rokwire/rokwire-building-blocks-api/issues/720)
 - Make catalog pages to show different items for reviewers and non reviewers. [#709](https://github.com/rokwire/rokwire-building-blocks-api/issues/709)
+- Make catalog pages to show all the items for contribution admins. [#721](https://github.com/rokwire/rokwire-building-blocks-api/issues/721)
```

Comment:
There doesn't seem to be an issue with number 721. I think this needs to be updated.

Reply:
changed it to 728